lib/content-services/src/lib/tag/services/tag.service.ts

```diff
@@ -105,6 +105,24 @@ export class TagService {
         return observableRemove;
     }
 
+    /**
+     * Deletes a tag with tagId.
+     * This will cause the tag to be removed from all nodes.
+     * You must have admin rights to delete a tag.
+     *
+     * @param tagId of the tag to be deleted
+     * @returns Null object when the operation completes
+     */
+    deleteTag(tagId: string): Observable<any> {
+        const observableDelete = from(this.tagsApi.deleteTag(tagId));
+
+        observableDelete.subscribe((data) => {
+            this.refresh.emit(data);
+        });
```

Comment:
Use a `tap` pipe here instead of subscribing.

Reply:
Done